pkgs/test_api/lib/src/backend/declarer.dart

```diff
@@ -195,6 +215,11 @@ class Declarer {
       bool solo = false}) {
     _checkNotBuilt('group');
 
+    final fullName = _prefix(name);
+    if (!(_filterToSingleTestName?.startsWith(fullName) ?? true)) {
```

Comment:
Possibly worth a comment?

I also find the `?? true` wrapped in parens and then negated very difficult to parse. I would write a more verbose form.

Reply:
I'll need to use `!` for the verbose form since it's a field. That's why I wrote it this way originally. I can expand it with the `!` though.